README.md

```diff
@@ -184,30 +210,24 @@ ansible-playbook drop-databases.yml                                \
     --user       $user
 ```
 
-### Migrate or reset the database
+### Deploy or delete the secrets
+This is a step that allows to modify kubernetes secrets.
+In normal scenario, we don't needed to use them, only when secrets are updated or clear all component on the cluster.
```

Comment:
Please reword:

```` markdown
### Deploying secrets
Before you can deploy containers, you need to make sure that certificates, keys and passwords used to configure those containers are available on the cluster.
Deploy them with:

``` bash
cd concent-deployment/cloud/
ansible-playbook cluster-deploy-secrets.yml                        \
    --extra-vars cluster=$cluster                                  \
    --inventory  ../../concent-deployment-values/ansible_inventory \
    --user       $user
```
This step is necessary only when deploying the cluster for the first time or when the secrets change.
Secret deployment is separate from deployment of the application specifically so that they can be performed separately, possibly from different machines.
````

Info about removing is not necessary. It's not a part of the usual workflow.

BTW, please keep a space between ```` ``` ```` and `bash`.

Reply:
Done.